Sources/Purchasing/EntitlementInfo.swift

```diff
@@ -161,6 +161,9 @@ extension PeriodType: DefaultValueProvider {
      */
     @objc public var ownershipType: PurchaseOwnershipType { self.contents.ownershipType }
 
+    /// Whether this entitlement was validated.
```

Comment:
This makes me think of a boolean. Could we link to the `EntitlementValidation` type?

Reply:
`Docc` will automatically have a link on the return type, but we could add
```swift
/// ### Related Symbols
/// - ``EntitlementValidation``
```

@aboedo any thoughts on whether that's overkill?

Reply:
Never mind, I see that I added this to `CustomerInfo`, so adding it here too 👍🏻